pinot-controller/src/test/java/com/linkedin/pinot/controller/helix/ControllerTest.java

```diff
@@ -49,6 +54,7 @@
  */
 public abstract class ControllerTest {
   public static final String LOCAL_HOST = "localhost";
```

Comment:
Is this used outside of this class? Otherwise, let's change it to `private`

Reply:
Its from previous version of the code; will fix it in a followup PR; will merge the current changes.